packages/terra-graphs/src/components/Line/LineGraph.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import Carbon from '../../../../carbon-graphs/src/js/carbon';
+import '../../css/Graph.module.scss';
+import '../../css/LineGraph.module.scss';
+import utils from '../../../../carbon-graphs/src/js/helpers/utils';
+
+class LineGraph extends React.Component {
+  render() {
+    debugger;
+    return (
+      <div id="canvasContainer">
+        <div id="tooltip" className="tooltip" style={{ display: 'none' }} />
```

Comment:
is this supposed to default to being hidden? we're using sass for component styles, so best to move inline styles there, then apply a specific css class via className

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/74/commits/631c1818a81503e72ba2f63e2f41eb9df667ba04#diff-3df99a036edf4bc1ff6a4da7270b378e91ed6043388577b14c68cf9f75d52f68R557-R559).